index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      #nav-bar {
+        position: fixed; 
+        top: 0; left: 0; width:100%;
+      }
+      @media(max-width:590px){
+        body {
+          font-size:75%;
+        }
+      }
+      .flex{
+        display:flex;
+        flex-direction: column;
+        flex-wrap: nowrap; 
+        align-content: center;   }
+    </style>
```

Comment:
Please ensure that you create a stylesheet for the css file for better code practice and readability.